ansible/files/ocp/ovn/northd.yaml

```diff
@@ -17,22 +17,22 @@ spec:
       containers:
       # ovn-northd: convert network objects in nbdb to flows in sbdb
       - name: northd
-        image: "quay.io/mbooth/ovn-central:2.11"
+        image: "quay.io/mbooth/ovn-central:trunk"
```

Comment:
As this gets polished you might just go ahead and push it to quay.io/openstack-k8s-operators?

Reply:
Yes, absolutely.